frontend/src/app/components/wp-buttons/wp-status-button/wp-status-button.component.ts

```diff
@@ -37,6 +37,8 @@ import {WorkPackageCacheService} from "core-components/work-packages/work-packag
 import {UntilDestroyedMixin} from "core-app/helpers/angular/until-destroyed.mixin";
 import {concatAll} from "rxjs/operators";
 import {combineLatest} from "rxjs";
+import {SchemaCacheService} from "core-components/schemas/schema-cache.service";
+import {ISchemaProxy, SchemaProxy} from "core-app/modules/hal/schemas/schema-proxy";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/3f0519806a21698d886f0aae816d7a75ed1548e8/frontend/src/app/components/wp-buttons/wp-status-button/wp-status-button.component.ts#L42"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>